libs/community/langchain_community/vectorstores/chroma.py

```diff
@@ -798,7 +798,7 @@ def delete(self, ids: Optional[List[str]] = None, **kwargs: Any) -> None:
         Args:
             ids: List of ids to delete.
         """
-        self._collection.delete(ids=ids)
+        self._collection.delete(ids=ids, **kwargs)
```

Comment:
delete lets you pass where and where_document, but wouldn't passing additional kwargs cause an error? @baskaryan